packages/devtools_app/lib/src/inspector/inspector_tree_controller.dart

```diff
@@ -211,27 +223,26 @@ class InspectorTreeController extends Object
     if (cachedRows[index]?.isSelected == true) {
       _cachedSelectedRow = cachedRows[index];
     }
-    return cachedRows[index];
+    final cachedRow = cachedRows[index];
```

Comment:
nit: move this up before the previous if block so the local var can be used in that if statement as well.